src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
move Random to the class field, like:
![image](https://user-images.githubusercontent.com/12864128/220328721-f24bbd91-9f17-489a-a49f-ba58323f4ee1.png)